modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.cs

```diff
@@ -427,10 +427,14 @@ static void LookupScriptForClass(Type type)
             // This method may be called before initialization.
             if (NativeFuncs.godotsharp_dotnet_module_is_initialized().ToBool() && Engine.IsEditorHint())
             {
-                foreach (var scriptPath in _pathTypeBiMap.Paths)
+
+                if (_pathTypeBiMap.Paths.Count > 0)
                 {
-                    using godot_string nativeScriptPath = Marshaling.ConvertStringToNative(scriptPath);
-                    NativeFuncs.godotsharp_internal_editor_file_system_update_file(nativeScriptPath);
+                    string[] script_paths = _pathTypeBiMap.Paths.ToArray();
```

Comment:
The naming convention for local variables in C# is `camelCase`.

```suggestion
                    string[] scriptPaths = _pathTypeBiMap.Paths.ToArray();
```